app/views/angular_template/test.html

```diff
@@ -0,0 +1,2 @@
+<!-- Please do not remove, will break tests :) -->
+It works!
```

Comment:
I see your point, but.. do we have any alternatives? Maybe mocking the template loading mechanism but I'm not quite sure whether that is better or worse..